### PR TITLE
Plane: add plane records

### PR DESCRIPTION
**Pull request by peterbarker**
The numbers in here are completely made up - I'm not even sure these are the true records.

### CODE REVIEW

**Comment by hendjoshsr71** [comment]
I really like this idea. But maybe it should be for more than just plane and a link prominent on the Wiki to it? Basically to show off it can handle challenging problems both in breadth and depth. 

Oh and maybe a smallest/lightest category too.

**Comment by peterbarker** [comment]
> Oh and maybe a smallest/lightest category too.

How do we measure "smallest"?  Wingspan?  Displacement?

**Comment by hendjoshsr71** [comment]
> > Oh and maybe a smallest/lightest category too.
> 
> How do we measure "smallest"? Wingspan? Displacement?

Can i punt and say both? ;) If I had to chose one do wingspan.

**Comment by MichelleRos** [comment]
Great idea 👍

**Comment by Hwurzburg** [comment]
Be sure to get permission from the record holder first.... Some might not be legal and they do not wish them published further

**Comment by peterbarker** [comment]
> Be sure to get permission from the record holder first.... Some might not be legal and they do not wish them published further

I'll do that when populating this for real.

Seems to be general agreement that this would be nice.

Any idea where we would link to this from elsewhere?

**Comment by peterbarker** [comment]
I've added a note into here with instructions on how to update, a note to check legality of flight, and a note that we won't accept any flights which would violate the Dev CoC...

File: plane/source/docs/records.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
++-------------------------+-----------------------+--------------+----------------+--------------------------------------------------+
++ Lightest                | ?                     +              +                +                                                  +
++-------------------------+-----------------------+--------------+----------------+--------------------------------------------------+
++ Longest                 | Krauss Aerospace      + 26 hours     +                +                                                  +
++-------------------------+-----------------------+--------------+----------------+--------------------------------------------------+
++ Furthest                | ?                     + km           +                +                                                  +
++-------------------------+-----------------------+--------------+----------------+--------------------------------------------------+
++ Higest $ Value Delivery | ?                     + US$          +                +                                                  +
```
Comment (hendjoshsr71) [inline plane/source/docs/records.rst:34]:
```suggestion
+ Highest $ Value Delivery | ?                     + US$          +                +                                                  +
```

Comment (peterbarker) [reply plane/source/docs/records.rst]: Done

**Pull request merged**